src/accidental.ts

```diff
@@ -62,7 +82,7 @@ export class Accidental extends Modifier {
         const accLine = Math.round((y / lineSpace) * 2) / 2;
         accList.push({ y, line: accLine, shift: shiftL, acc, lineSpace });
       } else {
-        accList.push({ line: props.line, shift: shiftL, acc });
+        accList.push({ line: props.line, shift: shiftL, acc } as AccidentalListItem);
```

Comment:
Which member of `AccitdentalListItem` is missing? Same comment as with `List` below.